.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,86 @@
+# As much as possible, this file should be kept in sync with
+# https://github.com/napari/docs/blob/main/.github/workflows/build_and_deploy_docs.yml
+name: Build PR Docs
+
+on:
+  pull_request:
+    types:
+        - opened
+        - synchronize
+  workflow_dispatch:
+    inputs:
+      make_target:
+        description: "Enter make target: html html-noplot docs slimfast slimgallery"
+        type: string
+        default: 'slimfast'
+  workflow_call:
+    inputs:
+      make_target:
+        description: "Enter make target: html html-noplot docs slimfast slimgallery"
+        type: string
+        default: 'slimfast'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-and-upload:
+    name: Build & Upload Artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone docs repo
+        uses: actions/checkout@v4
+        with:
+          # place in '/home/runner/work/docs/docs/docs'
+          path: docs  # place in a named directory
+
+      - name: Clone main repo
+        uses: actions/checkout@v4
+        with:
+          # place in '/home/runner/work/docs/docs/napari'
+          path: napari  # place in a named directory
+          repository: napari/napari
+          # ensure version metadata is proper
```

Comment:
Please clarify.

Reply:
FYI This yml is from napari/napari rather than new in this PR. In fact it used to exist in this repo first, but was removed in a cleanup. Alas it means it lost history and blame.
Anyhow, we used to have issues with the version displayed in the docs when the version was read in docs conf.py. This fixed it.

Reply:
@psobolewskiPhD Thanks. Let's put more context in the comment. Maybe "Fetch napari's version metadata history to accurately reflect docs version"

Reply:
So any changes to this yml should also be made in the other repo so they are in sync.

Reply:
done